runtime/vm/jvminit.c

```diff
@@ -2133,27 +2138,33 @@ VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved)
 
 					if (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_ENABLE_AOT)) {
 #if (defined(J9VM_ARCH_X86) || defined(J9VM_ARCH_S390) || defined(J9VM_ARCH_POWER) || defined(J9VM_ARCH_AARCH64))
-						if (argIndexXXPortableSharedCache > argIndexXXNoPortableSharedCache) {
-							vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE;
-						} else if (argIndexXXPortableSharedCache == argIndexXXNoPortableSharedCache) {
-							/* both "-XX:+PortableSharedCache" and "-XX:-PortableSharedCache" were not found thus following the default behavior for portable shared cache */
-							OMRPORT_ACCESS_FROM_J9PORT(vm->portLibrary);
-							BOOLEAN inContainer = omrsysinfo_is_running_in_container();
-							/* by default, enable portable shared cache in containers and disable otherwise */
-							if (TRUE == inContainer) {
-								vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE;
-							}
-						}
+						processPortableSharedCacheFlag = TRUE;
 #endif /* defined(J9VM_ARCH_X86) || defined(J9VM_ARCH_S390) || defined(J9VM_ARCH_POWER) || defined(J9VM_ARCH_AARCH64) */
 					}
 				}
-#if (defined(J9VM_ARCH_X86) || defined(J9VM_ARCH_S390) || defined(J9VM_ARCH_POWER) || defined(J9VM_ARCH_AARCH64)) && defined(J9VM_OPT_JITSERVER)
-				// Regardless of whether or not the local SCC is enabled, we still need to enable portable AOT if
-				// the options suggest we're going to connect to a JITServer AOT cache.
-				if (argIndexXXJITServerAOTCache > argIndexXXNoJITServerAOTCache) {
-					vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE;
+#if defined(J9VM_OPT_JITSERVER)
+				if ((argIndexXXJITServerAOTCache > argIndexXXNoJITServerAOTCache) &&
+				    (argIndexXXJITServerAOTCacheIgnoreLocalSCC > argIndexXXNoJITServerAOTCacheIgnoreLocalSCC)) {
+					// If we are using the JITServer AOT cache and ignoring the local SCC (which we detect by checking
```

Comment:
This is a .c file and it should use c style comments not c++.
Also, I think that VM coding style wants the `&&` to be at the beginning of the next line.

Reply:
Fixed.